UnitTests/Schemas/IFC4X3_RC4/viennese-bend_100.0_300_1000_1_Meter/src/viennese-bend_100.0_300_1000_1_Meter.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <EarlyBinding/IFC4X3_RC4/src/reader/IFC4X3_RC4Reader.h>
+#include <namespace.h>
+
+#include <IfcVisualTest.h>
+
+#include <IfcGeometryConverter/ConverterBuw.h>
+#include <IfcGeometryConverter/IfcImporter.h>
+#include <IfcGeometryConverter/IfcImporterImpl.h>
+
+using namespace testing;
+
+class VienneseBend : public IfcVisualTest {
+protected:
+
+	// Test standard values
+	buw::Image4b _background = buw::Image4b(0, 0);
+
+	virtual void SetUp() override {
+                IfcVisualTest::SetUp();
+
+		express_model = OpenInfraPlatform::IFC4X3_RC4::IFC4X3_RC4Reader::FromFile(filename.string());
+
+		importer = buw::makeReferenceCounted<oip::IfcImporterT<emt::IFC4X3_RC4EntityTypes>>();
+		auto model = importer->collectData(express_model);
+
+		_background = rendererIfc->captureImage();
+		rendererIfc->setModel(model[0]);
+	}
+
+	virtual void TearDown() override {
+		express_model.reset();
+                IfcVisualTest::TearDown();
+	}
+
+	virtual std::string TestName() const { return "viennese-bend_100.0_300_1000_1_Meter"; }
+	virtual std::string Schema() const { return "IFC4X3_RC4"; }
+
+	const boost::filesystem::path filename = dataPath("viennese-bend_100.0_300_1000_1_Meter.ifc");
+
+	std::shared_ptr<oip::EXPRESSModel> express_model = nullptr;
+	buw::ReferenceCounted<oip::IfcImporterT<emt::IFC4X3_RC4EntityTypes>> importer = nullptr;
+};
+
+TEST_F(VienneseBend, AllEntitiesAreRead) {
+	EXPECT_THAT(express_model->entities.size(), Eq(163));
+}
+/*
+TEST_F(VienneseBend, IFCHasAnEssentialEntity) {
+	auto result = std::find_if(express_model->entities.begin(), express_model->entities.end(), [](auto &pair) -> bool { return pair.second->classname() == "IFCTRIANGULATEDFACESET"; });
+	EXPECT_NE(result, express_model->entities.end());
+}
```

Comment:
Same as for cubic (alignmentHor with segement)

Reply:
Resolved ea463d4